### ISSUE TITLE
Remove the hardcoded column selection

### ISSUE THREAD

**Issue by DerickMathew** [DerickMathew/excel-and-snapsheets#8]
The snap2snapsheet.py has the following code

```
# TODO: Dont convert all the way to AAA, calculate it !!!
worksheet.set_column('A:AAA', 3)
```

Dont hardcode the AAA calculate it

**Comment by DerickMathew** [comment]
fixed in  7cfdc2c

**Comment by DerickMathew** [comment]
fixed in https://github.com/DerickMathew/excel-and-snapsheets/commit/7cfdc2cfa43e19e648e3c0dccf95a1c21700d03d
